.github/workflows/build.yml

```
@@ -528,12 +528,21 @@ jobs:
 
       - name: Rearchive and split artifact
         run: |
-          # compressed to artifact.7z.001, artifact.7z.002, ...
-          7z -r -v1g a "${{ matrix.artifact_name }}.7z" "${{ matrix.artifact_name }}/"
+          # Compress to artifact.7z.001, artifact.7z.002, ...
+          7z -r -v2g a "${{ matrix.artifact_name }}.7z" "${{ matrix.artifact_name }}/"
+          
+          # Compress to artifact.001.vvpp
```

細かいですが1つの場合は`artifact.vvpp`なので、こんな感じでしょうか。
```suggestion
          # Compress to artifact.vvpp or artifact.001.vvpp, artifact.002.vvpp, ...
```